### PR TITLE
Put the network entry from Azure CSV into the cloud cost

### PR DESCRIPTION
**Pull request by avrodrigues5**
## What does this PR change?
* Put the CSV line entry that satisfy the condition of category = network and meterCategory= Bandwidth  that represent the cost for egress out of the AKS cluster into cloudCost.
* THis is later used to reconcile the Network charges into Asset via cloud cost.

## Does this PR relate to any other PRs?
* Corresponding KCM PR

## How will this PR impact users?
* They will finally get to see a Network entry for a VMSS in cloud cost.
Screen Shot 2023-07-05 at 10 05 50 AM

## Does this PR address any GitHub or Zendesk issues?
* Closes ... [JIRA](https://kubecost.atlassian.net/browse/BURNDOWN-157)
Corresponding [github issue](https://github.com/kubecost/cost-analyzer-helm-chart/issues/2386)

## How was this PR tested?
* On Azure Cluster More details on KCM PR

## Does this PR require changes to documentation?
* No

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
* v1.105

### CODE REVIEW

**Review by nikovacevic — COMMENTED** [review]
Will wait for approval on https://github.com/kubecost/kubecost-cost-model/pull/1593, but this looks fine to me.

**Review by nikovacevic — APPROVED** [review]

**Comment by avrodrigues5** [comment]
@nikovacevic , This PR and cherry-pick PR https://github.com/opencost/opencost/pull/2024 needs to be merged!

**Pull request merged**
